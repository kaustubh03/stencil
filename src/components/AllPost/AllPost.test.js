import React from "react";
import renderer from "react-test-renderer";
import Allpost from ".";
import configureStore from "../../redux/configureStore";
const store = configureStore();
it("Allpost: default", () => {
  const component = renderer.create(<Allpost store={store} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
