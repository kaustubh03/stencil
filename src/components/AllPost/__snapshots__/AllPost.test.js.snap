// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Allpost: default 1`] = `
<div>
  <h1
    className="all_post_heading"
  >
    All Posts
  </h1>
</div>
`;
